keyboards/mechlovin/infinity875/rev1/info.json

```
@@ -0,0 +1,80 @@
+{
+    "usb": {
+        "pid": "0x0875",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
```

migrated from `rules.mk`
```suggestion
    "matrix_pins": {
        "custom_lite": true,
```

---

need to specify matrix pin/col when moving custom_lite to data driven. 

eg: 
```
☒ mechlovin/infinity875/rev1: MATRIX_COLS is inconsistent with the size of MATRIX_COL_PINS: 0 != 17
☒ mechlovin/infinity875/rev1: MATRIX_ROWS is inconsistent with the size of MATRIX_ROW_PINS: 0 != 6
```